osafw-app/App_Data/template/common/icons/alarm-fill.html

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-alarm-fill" viewBox="0 0 16 16">
+<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" class="bi bi-alarm-fill" viewBox="0 0 16 16">
```

Comment:
well, "16" used in bs5 icons original svg code, if we going to use "1em", then any updates (new icons) need to apply this change too

Reply:
Okay, I reverted that commit and added a CSS rules to the `site.css`. As BS icons are always squared it'll work. In other SVG icons fonts/sets there could be rectangular shapes and in that case every icon should have it's own width/height proportion according to the view box, like:

`width="0.875em" height="1em" viewBox="0 0 448 512"`